lib/pe_build/config/pe_agent.rb

```diff
@@ -51,6 +57,7 @@ def finalize!
     @autosign      = (not @master_vm.nil?) if @autosign  == UNSET_VALUE
     @autopurge     = (not @master_vm.nil?) if @autopurge == UNSET_VALUE
     @version       = 'current' if @version == UNSET_VALUE
+    @certname      = 'vm_name' if @certname = UNSET_VALUE
```

Comment:
I just saw how validation and defaults are intended to be done. I'll update my PR when I get a chance